paystell-frontend/src/app/payment-link/page.tsx

```diff
@@ -0,0 +1,133 @@
+import { Copy, Code } from "lucide-react";
+import { FaFacebook, FaWhatsapp, FaLinkedin } from 'react-icons/fa';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXTwitter } from '@fortawesome/free-brands-svg-icons';
+
+const PaymentLinkShare = () => {
+  const Link = "https://link.paystell.com/11aa22";
+  const embedCode = `<script src="https://integrate.paystell.com/" id="paystell-script" data-link="${Link}"></script>`;
+
+  const [linkCopyMessage, setLinkCopyMessage] = useState("");
+  const [embedCopyMessage, setEmbedCopyMessage] = useState("");
+  
+
+  const copyToClipboard = (text: string, setMessage: (message: string) => void) => {
+    if (navigator.clipboard && navigator.clipboard.writeText) {
+      navigator.clipboard.writeText(text).then(() => {
+        setMessage("Copied!");
+        setTimeout(() => {
+          setMessage("");
+        }, 2000);
+      }).catch(err => {
+        console.error('Failed to copy: ', err);
+        setMessage("Failed to copy");
+        setTimeout(() => {
+          setMessage("");
+        }, 2000);
+      });
+    } else {
+      const textArea = document.createElement("textarea");
+      textArea.value = text;
+      document.body.appendChild(textArea);
+      textArea.focus();
+      textArea.select();
+      try {
+        const success = document.execCommand('copy');
+        if (success) {
+          setMessage("Copied!");
+        } else {
+          setMessage("Failed to copy");
+        }
+      } catch (err) {
+        console.error('Fallback: Oops, unable to copy', err);
+        setMessage("Failed to copy");
+      }
+      document.body.removeChild(textArea);
+      setTimeout(() => {
+        setMessage("");
+      }, 2000);
+    }
+  };
+
+  return (
+    <>
+      <div className="container mx-auto px-4 py-6">
+        {/* Payment Link Section */}
+        <div className="p-6 mx-auto bg-white rounded shadow-md mb-6 max-w-full flex flex-col gap-4" style={{ width: '100%', maxWidth: '1117px', borderRadius: '8px', textAlign: 'left' }}>
+          <h1 className="text-xl font-bold mb-4" style={{ fontSize: '24px' }}>Now share your payment link as you prefer</h1>
+          <div className="mb-4 flex flex-col md:flex-row items-start md:items-center space-y-2 md:space-y-0 md:space-x-2">
+            <input
+              type="text"
+              value={Link}
+              readOnly
+              className="w-full md:w-3/5 border px-4 py-2 rounded-lg focus:outline-none"
+              style={{ userSelect: 'none' }}
+            />
+            <button
+              onClick={() => copyToClipboard(Link, setLinkCopyMessage)}
+              className="w-full md:w-auto bg-[#009EFF] text-white px-4 py-2 rounded-lg flex items-center justify-center space-x-2 transition-transform transform hover:scale-105 active:scale-95"
+            >
+              <Copy className="w-4 h-4" />
+              <span>Copy Link</span>
+            </button>
```

Comment:
please use Button component located in components/ui for all buttons 
```suggestion
              <Button
                type="button"
                variant="default"
                onClick={() => copyToClipboard(Link, setLinkCopyMessage)}
              >
              <Copy className="w-4 h-4" />
              Copy Link
              </Button>
            
```